### ISSUE TITLE
harmonizing more than two datasets

### ISSUE THREAD

**Issue by johnaeanderson** [nancynewlin-masi/MidRISH#1]
Thank you for this very thorough set of code and for the approach - I agree that selecting a midpoint makes more sense than choosing one site as a reference. My question is what happens if we want to harmonize multiple sites? Should this "just work" assuming we have matched the individuals and that there is equal representation from each site for the template creation?

**Comment by nancynewlin-masi** [comment]
Hello, thank you for your interest in this project! Moving from the N=2 case to N>2 is a bit more complicated to code. We included a formula for these cases in our paper (formula attached), however we did not make the associated code implementation for N>2. 
But yes, that is the idea; We would adjust the formula from k=2 to k=N and include representation from all sites and compute some "center" or "mid" space that is the average of each site's coefficients. 
<img width="232" alt="image" src="https://github.com/user-attachments/assets/daa7a19a-294c-4e84-8b84-76cba81b5ae4">

Link: https://www.sciencedirect.com/science/article/abs/pii/S0730725X24000894
